zjit/src/hir.rs

```diff
@@ -3797,7 +3779,12 @@ impl Function {
                     String::new()
                 };
 
-                let opcode = insn.print(&ptr_map).to_string();
+                // For parameters, use numbered label. For other instructions, use the opcode.
```

Comment:
Why do we want or need this branch? I think it's fine if they say "Param"; they are named by their InsnId and that is already in the output

Reply:
Let's remove it